src/app/loadout-builder/filter/TierSelect.tsx

```diff
@@ -77,22 +94,25 @@ export default function TierSelect({
                   </span>
                 }
               >
+                <span className={styles.range}>
+                  {t('LoadoutBuilder.MaxTier', {
+                    tier: t('LoadoutBuilder.TierNumber', {
+                      tier: statTierWithHalf(statRangesFiltered[statHash].max),
```

Comment:
```suggestion
                      tier: statTierWithHalf(statRanges[statHash].max),
```

Reply:
If it is around the right way then I just think its the max thing is confusing as @inexorableAce pointed out. Really atm the "Max" is the max of the sets returned from process..

Reply:
I hope I've managed to explain this below, but no, I don't think we should use `statRanges`.